src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -585,6 +585,7 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                         ->scalarNode('version')->defaultNull()->end()
                         ->scalarNode('version_format')->defaultValue('%%s?%%s')->end()
                         ->scalarNode('json_manifest_path')->defaultNull()->end()
+                        ->booleanNode('cache_version')->defaultFalse()->end()
```

Comment:
why not `defaultNull`?

Reply:
`use_cache` => true/false/string => when string, it's the name of the cache pool to use?

Reply:
(or just `cache` btw)

Reply:
I still don't get the name `cache_version`
`cache: true/false` looks better to me. Do I miss something?

Reply:
I've renamed to `cache`.
I wanted to avoid confusion. This cache the result of the URL generation, a developer might think this is the HTTP cache when the browser requests the URL.

Reply:
`cache_json_manifest` then?

Reply:
or `json_manifest_cache` might even be better?

Reply:
It can cache any version strategy, not only the manifest.
We could also remove this option and always enable the cache on remote json manifests?

Reply:
Can we? How would this work with env vars?
Or we might just always enable the cache, even for local files.
For thoughts :)